docs/cec-mqtt-bridge.md

```diff
@@ -0,0 +1,12 @@
+# HDMI-CEC-MQTT-bridge
+## References
+- [Docker](https://hub.docker.com/r/jonaseck/rpi-cec-mqtt-bridge)
+- [Upstream documentation](https://github.com/michaelarnauts/cec-mqtt-bridge/blob/master/README.md)
+- [cec-o-matic](http://www.cec-o-matic.com/)
+
```

Comment:
Hey @DIYtechie do you mind adding in a sentence or 2 on what cec-mqtt-bridge is? It can be copied from their webpage/github.